SCPlayer/SCPlayer/Controllers/Cores/Home Screen/HomeViewController.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  HomeViewController.swift
+//  SCPlayer
+//
+//  Created by Thuận Nguyễn Văn on 10/05/2021.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
```

Comment:
```suggestion
final class HomeViewController: UIViewController {
```
mấy class view khác cũng final hết nhé

Reply:
Oke. thanks banj nha